src/openforms/forms/tests/test_serializers.py

```diff
@@ -0,0 +1,36 @@
+from hypothesis import given
+from hypothesis.extra.django import TestCase as HypothesisTestCase
+
+from openforms.forms.api.datastructures import FormVariableWrapper
+from openforms.forms.api.serializers.logic.action_serializers import LogicComponentActionSerializer
+from openforms.forms.tests.factories import FormFactory, FormVariableFactory
+from openforms.tests.search_strategies import json_values
+from openforms.variables.constants import FormVariableDataTypes, FormVariableSources
+
+
+class LogicComponentActionSerializerPropertyTest(HypothesisTestCase):
+    @given(json_values())
+    def test_date_format_validation_against_arbitrary_json_values(self, json_value):
```

Comment:
can you add a happy flow test so that a date type variable still works with a non-trivial json logic expression like `{"var": "anotherDate"}`?